tests/component-tests/pager/js/PagedTableTests.js

```diff
@@ -160,30 +160,22 @@ https://github.com/fluid-project/infusion/raw/master/Infusion-LICENSE.txt
 
     fluid.tests.tooltipModuleSource = function (pager) {
         var sequence = [];
+        var linksList = pager.dataModel.map(function(data) {
+            return {text: data.animal};
+        });
+
+        /* Change the page size and verify whether the tooltips and links are synced accordingly */
+        for (var pageSize = 2; pageSize <= 4; pageSize++) {
```

Comment:
maybe you can take the pageSize range as an argument in `fluid.tests.tooltipModuleSource`